docs/_data/prague-2022-schedule.yaml

```diff
@@ -0,0 +1,126 @@
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '0:15'
+    title: 'Coffee break'
+  - duration: '0:30'
+    slug: documentation-as-marketing-from-conflict-to-collaboration-mano-toth
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: cultivating-a-stakeholder-network-for-our-docs-how-building-relationships-impro-taylor-cole
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '1:00'
+    title: 'Lunch Break'
+  - duration: '0:30'
+    title: Lightning Talks (30 mins)
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: getting-started-and-harnessing-the-opportunities-in-technical-writing-deborah-emeni
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: beyond-spell-checking-what-else-can-we-check-automatically-tibs
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '0:00'
+    title: Day 1 wraps up
+  - time: '17:00'
+    title: "<b>Social Event</b>"
+  - time: '19:00'
+    title: "<b>Social Event Ends</b>"
+
+talks_day2:
+  - time: '8:00'
+    title: Platform opens
+    duration: '1:00'
+  - duration: '0:00'
+    title: "<b>Job Fair Starts</b>"
+  - duration: '0:30'
+    slug: the-art-of-asking-questions-jeff-smith
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: improve-customer-adoption-with-ui-help-prashant-mathapathi
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '0:15'
+    title: 'Coffee break'
+  - duration: '0:30'
+    slug: toward-the-broader-globalization-of-open-source-documenting-your-localisation-j-batool-almarzouq
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:25'
+    title: "<b>Meetup information session - Get involved locally!</b>"
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:15'
+    title: '✋ Q&A for the previous talk'
+  - duration: '0:00'
+    title: "<b>Job Fair Ends</b>"
+  - duration: '1:00'
+    title: 'Lunch Break'
+  - duration: '0:30'
+    title: Lightning Talks (30 mins)
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:00'
+    title: "<b>Unconference Starts</b>"
+  - duration: '0:30'
+    slug: two-years-of-markdoc-what-we-ve-learned-about-balancing-developer-and-author-ex-ryan-paul
+  - duration: '0:05'
+    title: '5 minute break'
+  - duration: '0:30'
+    slug: creating-documentation-for-the-african-audience-benny-ifeanyi-iheagwara
+  - duration: '0:15'
+    title: 'Closing address'
+  - duration: '0:15'
+    title: '✋ Q&A for the two previous talks'
+  - duration: '0:20'
+    title: '<b>Main stage talks done</b>'
+  - duration: '1:30'
+    slug: adhd-and-its-impact-on-the-creative-process-for-writers-tyler-smith
```

Comment:
Why is this after the main stage talks done? Looks like a copy-paste error

Reply:
Yeah, that's the bit we need to reorganize, it was the Google Workshop 

> OK, now we're using the updated template, we should probably fix the two isolated talks on Tuesday, so it's 2 and 2, not 2 1 1, right?

4 talks in the morning, 2 in the afternoon with a later lunch?

Reply:
We're probably also not running this? 

> Meetup information session - Get involved locally!

Reply:
Yeah let's clean up the workshop and whatever else isn't the talks that we selected, and if anything gets added back in then we'll add it :)